src/app/components/big-input/big-input.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="container">
+  <input class="input" type="search" placeholder="Smith" />
+</div>
```

Comment:
It is necessary?
```suggestion
<input class="input" type="search" placeholder="Smith" />
```

Reply:
Because the input tag isn't available to have children elements, I had problems adding the search icon as a pseudoelement. At the end I just created the div container to do that. And in the future we need to add more elements to this input to manage the selectors of gender and status, so I did  it in that way. 

Do you know a better way to do it?